configs/meteo-france/config.yaml

```diff
@@ -43,40 +43,26 @@ website:
     display: true
     name: 'Recherche guidée'
     link: '/form'
-  router_items:
-    - name: 'Accueil'
-      id: 'home'
-      linkPage: '/'
-      type: 'component'
-      display_menu: false
-    - name: 'Données'
-      id: 'datasets'
-      linkPage: '/datasets'
-      type: 'component'
-      display_menu: true
-    - name: 'Recherche guidée'
-      id: 'FormMF'
-      linkPage: '/form'
-      type: 'custom'
-      custom_component: 'meteo/views/FormMF'
-      display_menu: true
-    # - name: 'Portail des API Météo-France'
-    #   id: 'portailapi'
-    #   linkPage: 'https://portail-api.meteofrance.fr/'
-    #   type: 'external'
-    #   display_menu: true
-    - name: 'A propos'
-      id: 'about'
-      linkPage: '/about'
-      type: 'page'
-      url: '/meteo-france/pages/about.md'
-      display_menu: true
-    - name: 'Accessibilité'
-      id: 'accessibility'
-      linkPage: '/accessibility'
-      type: 'page'
-      display_menu: false
-      url: '/meteo-france/pages/accessibility.md'
+  menu:
+    - name: Données
+      path: /datasets
+    - name: Recherche guidée
+      path: /form
+    - name: A propos
+      path: /about
+  router:
+    disable:
+      - organizations_routes
+      - topics
+    pages:
+      - name: 'A propos'
```

Comment:
Pas fan de la duplication de `name` et `path`/`linkPage` entre `menu` et ici.
`menu` pourrait être une simple liste d'`id` non ?

Reply:
Je suis d'accord que c'est dommage mais ça me permet de gérer le cas lien externe https://github.com/opendatateam/udata-front-kit/blob/3f04646067e50194b23a90820ce0e732c29d4ddb/configs/defis/config.yaml#L68

On pourrait avoir :

```
interface MenuConfig {
  name: string
  href?: string // external
  to?: string // internal
}
```

Mais peut-être moins évident niveau config et un peu de code en plus pour parser les routes dans le cas `!!to`.

Reply:
Quant au `name` il n'existe pas sur une route (ou en tout cas ne veut pas dire la même chose).

Reply:
En effet le `name` n'est pas équivalent :eyes: 

IMO changer le nom de la propriété suffirait : 
- Dans `menu`, changer pour `label` ou `text`. 
- Dans `router_items`, changer pour `title`.

Dans les deux cas c'est plus cohérent avec l'usage qui en est fait dans le code, et ça reste tout aussi explicite. Dans le cas de `router_items`, ça évitera aussi le risque de noeud au cerveau avec le [switch id/name](https://github.com/opendatateam/udata-front-kit/blob/3f04646067e50194b23a90820ce0e732c29d4ddb/src/router/index.ts#L67C1-L75C4).

+1 sur `path` permettant indifférement les liens internes/externes, c'est pratique !

Reply:
nouvelle proposition https://github.com/opendatateam/udata-front-kit/pull/429/commits/d099b926801bfe79f0454d4bb25c74f347a671ce

Reply:
Nettement plus clair :)

Je viens de me rendre compte qu'on a déjà des trucs similaires dans la config au niveau du footer. IMO c'est l'occasion de standardiser (il restera quelques incohérences, mais c'est déjà ça de pris !).

Si on préfère éviter de toucher aux sections footer (pour réduire le diff de cette PR, sinon j'ai pas de préférence), ça voudrait dire : 
- `menu.text` -> `menu.label`.
- `menu.path` -> `menu.to` / `menu.href`. Comme tu le suggérais ci-dessus... Mais peut-être que là ça a un sens de standardiser dans l'autre sens sur `footer_*.path` ? (ici ou autre PR plus tard).